common/app/model/ShareLinks.scala

```diff
@@ -93,7 +93,7 @@ object ShareLinks {
     lazy val facebookParams = List(
       Some("app_id" -> facebookAppId),
       Some("href" -> encodedHref),
-      quote.map(q => "quote" -> q),
+      quote.map(q => "quote" -> q.urlEncoded),
       mediaPath.map(path => "picture" -> path.urlEncoded)
```

Comment:
something still isn't right.  We're calling a method on a uri builder to add parameters, and it's falling over because it's failing to escape them properly?
https://github.com/guardian/frontend/blob/bf6421b383c4eb7a0aba68a13e8bedfff35d0420/common/app/implicits/Strings.scala#L50
according to the docs, it should be escaping them
http://docs.oracle.com/javaee/7/api/javax/ws/rs/core/UriBuilder.html
> Builder methods perform contextual encoding of characters not permitted in the corresponding URI component following the rules of the application/x-www-form-urlencoded media type for query parameters and RFC 3986 for all other components. Note that only characters not permitted in a particular component are subject to encoding